### PR TITLE
Tesla tweaks and bugfixes

### PR DESCRIPTION
**Pull request by Sindorman**
- Bugfix: Tesla no longer gains energy from just colliding with objects

- Bugfix: Fixes sprite for singularity beacon not showing up

- Bugfix: Tesla no longer spams admin longs when a new ball is created outside of containment

- Bugfix: Telsa looses energy when dropping ball which should be slightly below how much is required for new ball. 

- tweak: Tesla dissipation rate has been buffed, it now looses energy faster

- tweak: Tesla dissipates every time it zaps something

- tweak: Tesla now melts any object or mob(but not turfs) it touches or when it zaps it while sacrificing a miniball.

- rscadd: Added special emergency singularity beacon that is to be used when Singulo/Telsa are on the Loose. When Tesla zaps the beacon, it will discharge all energy into it, dying and destroying beacon.

### CODE REVIEW

**Comment by Sindorman** [comment]
!review

File: code/modules/power/tesla/energy_ball.dm [hunk 1]
```diff
@@ -196,12 +199,30 @@
...
+	if(isliving(A))
+		dust_mobs(A)
+	else if(isobj(A))
+		var/obj/O = A
+		O.tesla_act(0, TRUE)
+
+/obj/singularity/energy_ball/Move(NewLoc, Dir)
 	. = ..()
-	dust_mobs(A)
+	for(var/v in view(0, loc))
```
Comment (skull132) [inline code/modules/power/tesla/energy_ball.dm]: Why not just: `for (var/v in loc)`

Comment (skull132) [reply code/modules/power/tesla/energy_ball.dm]: Also, why is this necessary; shouldn't collidedwith take care of this logic?

Comment (Sindorman) [reply code/modules/power/tesla/energy_ball.dm]: Because I want to ignore things under Turf. It eats all pipes/cables which is not what we want.

**Pull request merged**